Userland/Libraries/LibWeb/Painting/MarkerPaintable.cpp

```diff
@@ -69,6 +78,34 @@ void MarkerPaintable::paint(PaintContext& context, PaintPhase phase) const
     case CSS::ListStyleType::Disc:
         aa_painter.fill_ellipse(device_marker_rect.to_type<int>(), color);
         break;
+    case CSS::ListStyleType::DisclosureClosed: {
+        // https://drafts.csswg.org/css-counter-styles-3/#disclosure-closed
+        // For the disclosure-open and disclosure-closed counter styles, the marker must be an image or character suitable for indicating the open and closed states of a disclosure widget, such as HTML’s details element.
+        // FIXME: If the image is directional, it must respond to the writing mode of the element, similar to the bidi-sensitive images feature of the Images 4 module.
+
+        // Draw an equilateral triangle pointing right.
+        auto path = Gfx::Path();
+        path.move_to({ left, top });
+        path.line_to({ left + sin_60_deg * (right - left), (top + bottom) / 2 });
+        path.line_to({ left, bottom });
+        path.close();
+        aa_painter.fill_path(path, color);
```

Comment:
super nit: `fill_path()` defaults to a nonzero fill, but this path could be filled via evenodd (which _happens_ to be more efficient)